pkg/ui/src/views/databases/containers/databaseGrants/index.tsx

```diff
@@ -49,21 +75,28 @@ class DatabaseSummaryGrants extends DatabaseSummaryBase {
             <div className="database-summary-table sql-table">
               {
                 (numTables === 0) ? "" :
-                  <DatabaseGrantsSortedTable
-                    data={grants as protos.cockroach.server.serverpb.DatabaseDetailsResponse.Grant[]}
-                    sortSetting={sortSetting}
-                    onChangeSortSetting={(setting) => this.props.setSort(setting)}
-                    columns={[
-                      {
-                        title: "User",
-                        cell: (grant) => grant.user,
-                        sort: (grant) => grant.user,
-                      },
-                      {
-                        title: "Grants",
-                        cell: (grant) => grant.privileges.join(", "),
-                      },
-                    ]} />
+                  <React.Fragment>
+                    <DatabaseGrantsSortedTable
```

Comment:
@elkmaster Can we combine the `PaginationComponent` and `DatabaseGrantsSortedTable` into a `PaginatedDatabaseGrantsSortedTable`? See if it's straightforward to do. If it's complicated I'm ok with leaving as-is.

Extra challenge: can we create a generic `PaginatedTable` that's parametrized with the table type?

Reply:
If we create `PaginatedDatabaseGrantsSortedTable` it will look very similar to existing file `databaseGrants/index.tsx`. I would left this as-is, as we still will be restructuring table views, after we move\create table component in upcoming design system. Creating generic PaginatedTable that's parameterized, also make more sense doing after, and task big enough to have at least it own ticket.